airsonic-main/src/main/java/org/airsonic/player/service/MediaScannerService.java

```diff
@@ -69,13 +68,13 @@
 
     @PostConstruct
     public void init() {
-        deleteOldIndexFiles();
+        indexManager.deleteOldIndexFiles();
         statistics = settingsService.getMediaLibraryStatistics();
         schedule();
     }
 
     public void initNoSchedule() {
-        deleteOldIndexFiles();
+        indexManager.deleteOldIndexFiles();
```

Comment:
Same here, would prefer not to call `indexManager.deleteOldIndexFiles()` automatically.

Reply:
Same as above. Underlying is the issue of Airsonic's management policy.

Reply:
Let's comment this out for now.

Reply:
We have approved your request.